frontend/src/apis/experiment/api.ts

```diff
@@ -254,6 +254,7 @@ export interface ApiStatus {
  * @enum {string}
  */
 export enum ExperimentStorageState {
+  UNSPECIFIED = <any>'STORAGESTATE_UNSPECIFIED',
```

Comment:
nit: worth splitting this into a separate PR?

Reply:
thanks, it's auto generated and won't have UI usage change. I think it's fine.
/cc @jingzhang36